src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+
+    String by = "";
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    public String toString() {
+        return String.format("[D][ ] " + description + " (by: " + by + ")");
```

Comment:
Can I check if isDone status is implemented for printing of deadline and events?
It seems like the "getStatusIcon" method is only used in Todo